cmd/nodecmd/create.go

```diff
@@ -1188,6 +1197,50 @@ func requestCloudAuth(cloudName string) error {
 	return nil
 }
 
+func getSeparateHostNodeParam(cloudName string) (
+	string,
+	error,
+) {
+	type CloudPrompt struct {
+		defaultLocations []string
+		locationName     string
+		locationsListURL string
+	}
+
+	supportedClouds := map[string]CloudPrompt{
```

Comment:
it's the same as `getRegionsNodeNum` - we should move it out from both func so it can be DRY. I suggest moving it to cloud/aws and cloud/gcp as a constants.

another option is to change getRegionsNodeNum func to accomodate both cases when we want to accept slice or single value so no need to loop through prompts